dojo/tools/gitlab_container_scan/parser.py

```diff
@@ -69,28 +60,35 @@ def get_findings(self, file, test):
                 test=test,
                 description=description,
                 severity=severity,
-                cve=cve,
-                cwe=cwe,
                 static_finding=True,
                 dynamic_finding=False,
             )
 
             # Add component fields if not empty
+            if cve != "":
+                finding.cve = cve
+            if cwe != "":
+                finding.cwe = cwe
             if component_name != "":
-                finding.component_name = component_name[:190] + (component_name[190:] and '...')
+                finding.component_name = component_name[:190] + (
+                    component_name[190:] and "..."
+                )
```

Comment:
For this kind of shortening task, you can use `textwrap.shorten` see https://docs.python.org/3/library/textwrap.html#textwrap.shorten